tests/compile/test_aot_compile.py

```diff
@@ -0,0 +1,128 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Copyright contributors to the vLLM project
+
+import tempfile
+from contextlib import contextmanager
+
+import pytest
+import torch
+
+from vllm.compilation.decorators import support_torch_compile
+from vllm.config import (CompilationConfig, CompilationLevel, VllmConfig,
+                         set_current_vllm_config)
+from vllm.forward_context import set_forward_context
+
```

Comment:
NB: vLLM does run vLLM main x PyTorch nightly, but that CI is very busted (it's been failing for weeks), so we're not going to get signal that way. Zhengxu has verified that this test file passes locally, plus this change is opt-in and still undergoing integration testing.